ci/lint/04_install.sh

```diff
@@ -8,13 +8,19 @@ export LC_ALL=C
 
 ${CI_RETRY_EXE} apt-get update
 ${CI_RETRY_EXE} apt-get install -y python3-pip curl git gawk jq
-(
-  # Temporary workaround for https://github.com/bitcoin/bitcoin/pull/26130#issuecomment-1260499544
-  # Can be removed once the underlying image is bumped to something that includes git2.34 or later
-  sed -i -e 's/bionic/jammy/g' /etc/apt/sources.list
-  ${CI_RETRY_EXE} apt-get update
-  ${CI_RETRY_EXE} apt-get install -y --reinstall git
-)
+${CI_RETRY_EXE} apt-get install -y make build-essential libssl-dev zlib1g-dev \
+libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
+libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
```

Comment:
Assuming that those packages are required to build the Python, why they are installed every time, regardless of using of the cached Python?

The `curl` package was installed two lines above.

Why `llvm`, `xz-utils` and `libffi-dev` are required?

Reply:
The packages listed there is an exact copy as detailed by the pyenv docs. As I have said, my concern is making it as maintainable as possible